packages/host/app/utils/editor-language.ts

```diff
@@ -11,12 +11,12 @@ export type MonacoLanguageConfig = {
 };
 
 const gjsConfig: LanguageInfo = {
-  id: 'glimmerJS',
+  id: 'gjs',
   extensions: ['.gjs'],
 };
 
 const gtsConfig: LanguageInfo = {
-  id: 'glimmerTS',
+  id: 'gts',
```

Comment:
This change was needed so that monaco can understand glimmer language when the assistant responds with ` ```gts ` code snippets. Alternatively we could change the skill to use ` ```glimmerTS ` but I think the former solution sounds more standard